configs/katello/3.18.yaml

```diff
@@ -64,3 +71,25 @@
       - centos-sclo-rh-rhel-7
       - centos-7-server-updates
       - centos-7-server
+  - name: katello-3.18-el8
+    based_off: katello-nightly-el8
```

Comment:
There were already changes merged to nightly. How are we going to sync up the koji tags and git repo?

Reply:
Luckily, the changes are minimal:

```
% git diff --stat rpm/2.3..rpm/develop packages/katello/ 
 packages/katello/katello-repos/katello-repos.spec                           |  8 ++++----
 packages/katello/katello/katello.spec                                       |  8 ++++----
 packages/katello/rubygem-hammer_cli_katello/hammer_cli_katello-0.24.0.gem   |  1 -
 packages/katello/rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec |  8 +++++---
 packages/katello/rubygem-katello/katello-3.18.0.rc1.gem                     |  1 -
 packages/katello/rubygem-katello/rubygem-katello.spec                       | 21 +++++++++++----------
 6 files changed, 24 insertions(+), 23 deletions(-)
```

I'll just untag those packages and rebuild them from what's in 2.3.